Regression/Checksum/checksumAPI.py

```diff
@@ -80,7 +80,7 @@ def reset_all_benchmarks(path_to_all_plotfiles):
     '''
 
     # Get list of plotfiles in path_to_all_plotfiles
-    plotfile_list = glob.glob(path_to_all_plotfiles + '*_plt?????',
+    plotfile_list = glob.glob(path_to_all_plotfiles + '*_plt[0-9]+',
```

Comment:
This does not work.
Needs to be "literal last character is a number"
```suggestion
    plotfile_list = glob.glob(path_to_all_plotfiles + '*_plt*[0-9]',
```